client/my-sites/patterns/pages/home/index.tsx

```diff
@@ -51,16 +50,11 @@ export const PatternsHomePage = ( {
 				title="Build your perfect site with patterns"
 			/>
 
-			<PatternsGridGallery
+			<CategoryGallery
```

Comment:
Hm, I have a concern regarding `Category` word in the component name, I would call it more generic to avoid confusion with page `/patterns:category`. WDYT?

Reply:
Hmm, I guess we could rename it to something like `CategoryLinkGallery` or `CategoryLinkGrid`? The links here do lead to `/patterns/:category` pages, so I think mentioning "category" makes sense. But I agree that `CategoryGallery` can be confused for a gallery shown on the category page

Reply:
Thinking one more time - I got your point for this name. Ok, let's go with the current name, it's not so bad